CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
-# Contributing to Docker
+# Contributing to the Moby Project
 
-Want to hack on Docker? Awesome!  We have a contributor's guide that explains
-[setting up a Docker development environment and the contribution
+Want to hack on the Moby Project? Awesome! We have a contributor's guide that explains
+[setting up a development environment and the contribution
 process](https://docs.docker.com/opensource/project/who-written-for/). 
```

Comment:
Update this to live in the Moby repo now? Could just view it on Github directly.